doc/src/vpicUnits.tex

```diff
@@ -2,33 +2,47 @@
 
 \usepackage{amsmath}
 \usepackage{amsfonts}
+\usepackage{amssymb}
 \usepackage{geometry}
 
 \geometry{left=2cm,right=2cm,top=3cm,bottom=2.5cm}
 
 \renewcommand{\vec}[1]{\mathbf{#1}}
+\newcommand{\omegap}{\omega_{\mathrm{pe}}}
+\newcommand{\ncrit}{n_{\mathrm{cr}}}
+\newcommand{\vth}{v_{\mathrm{th}}}
+\newcommand{\ldebye}{\lambda_{\mathrm{D}}}
 
 \title{Units in VPIC}
 \author{A. Seaton}
 \begin{document}
 	\maketitle
 
-	Internally, the code uses the SI form for Maxwell's equations and the Lorentz force (i.e. not the Guassian form). However, the code allows the user to specify various constants, and these determine the unit system. In particular, the user implicitly defines a length scale $l_{\mathrm{deck}}$ for the spatial grid, and explicitly sets values for the speed of light ($c_{\mathrm{deck}}$), vacuum permittivity ($\varepsilon_{0,\mathrm{deck}}$) and charge/mass ratio of each species $\alpha$ ($r_{\alpha,\mathrm{deck}} \equiv (q_{\alpha}/m_{\alpha})_{\mathrm{deck}}$). Together these define the unit system used by the code.
+	Internally, the code uses the SI form for Maxwell's equations and the Lorentz force (i.e. not the Guassian form). However, the code requires the user to specify various constants, and these determine the unit system. In particular, the user defines:
 
-	One reason for this flexibility (versus having a fixed unit system) is that VPIC uses single-precision floating point numbers. This means that all values, including at intermediate steps in calculations, must have absolute value $|x|$ between $\sim 10^{-38}$ and $\sim 10^{38}$. Depending on the problem, a unit system such as SI may require values smaller or larger than these.
+	\begin{itemize}
+		\item $l_{\mathrm{deck}}$: an implicit unit of length for the spatial grid
+		\item $c_{\mathrm{deck}}$: the speed of light in vacuum
+		\item $\varepsilon_{0,\mathrm{deck}}$: the vacuum permittivity
+		\item $r_{\alpha,\mathrm{deck}} \equiv (q_{\alpha}/m_{\alpha})_{\mathrm{deck}}$: The charge/mass ratio of species $\alpha$.
+	\end{itemize}
+
+	\noindent Together these fully determine the unit system used by the code. One reason for this flexibility (versus having a fixed unit system) is that VPIC uses single-precision floating point numbers. This means that all values, including at intermediate steps in calculations, must have absolute value $|x|$ between $\sim 10^{-38}$ and $\sim 10^{38}$. Depending on the problem, a unit system such as SI may require values smaller or larger than these.
```

Comment:
I wouldn't say that VPIC itself has a fully determined unit system.  It just solves equations given some wave speed and field coupling on a user-defined grid.  You can't, for example, calculate the fine structure constant in the code without making assumptions about how the user is using VPIC.

Reply:
Yep I suppose I'm being a bit loose with the language. Does this sound better to you?
```suggestion
	\noindent Together these fully determine the user's unit system. One reason for this flexibility (versus having a fixed unit system) is that VPIC uses single-precision floating point numbers. This means that all values, including at intermediate steps in calculations, must have absolute value $|x|$ between $\sim 10^{-38}$ and $\sim 10^{38}$. Depending on the problem, a unit system such as SI may require values smaller or larger than these.
```